### PR TITLE
Add com.github.Alcaro.Flips

### PR DESCRIPTION
**Pull request by Kekun**
The source will have to be updated once https://github.com/Alcaro/Flips/pull/27 is accepted.

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build com.github.Alcaro.Flips

File: com.github.Alcaro.Flips.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        "--device=dri",
+        /* Needed to find the ROMs */
+        "--filesystem=host:ro"
+    ],
+    "build-options" : {
+        "cflags" : "-O2 -g",
+        "cxxflags" : "-O2 -g",
+        "env" : {
+        }
+    },
```
Comment (bilelmoussaoui) [inline com.github.Alcaro.Flips.json]: Already set in the runtime.

Comment (Kekun) [reply com.github.Alcaro.Flips.json]: Do you know if it's also set on master? So I can remove it whenever I see it elsewhere. :)

Comment (bilelmoussaoui) [reply com.github.Alcaro.Flips.json]: They are set on fdo, so in GNOME master too and everything that derives from it.

Comment (Kekun) [reply com.github.Alcaro.Flips.json]: Perfect, thanks! \o/

**Comment by bilelmoussaoui** [comment]
bot, build com.github.Alcaro.Flips

**Comment by Kekun** [comment]
There is no point building it yet, it won't work as it's not pointing to the right place but to master, which didn't accept the MR adding the install rule and more. :( To you want me to temporarily make it point to my repo and branch so it can build?

**Comment by bilelmoussaoui** [comment]
> There is no point building it yet, it won't work as it's not pointing to the right place but to master, which didn't accept the MR adding the install rule and more. :( To you want me to temporarily make it point to my repo and branch so it can build?

Yes please :)

**Comment by Kekun** [comment]
OK, done. :)

**Comment by bilelmoussaoui** [comment]
bot, build com.github.Alcaro.Flips

**Comment by Kekun** [comment]
It should work now.

**Comment by Kekun** [comment]
Oh, and if it works and you don't mind pointing to my "fork", maybe my repo can be used as the source until @Alcaro accepts my MR.

**Comment by LeandroStanger** [comment]
bot, build com.github.Alcaro.Flips

**Comment by Kekun** [comment]
All the relevant metadata *and* the source manifest have been accepted upstream, I just submitted the updated manifest here. Let me know if it's good. :)

**Comment by bilelmoussaoui** [comment]
bot, build com.github.Alcaro.Flips

File: com.github.Alcaro.Flips.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            "buildsystem" : "simple",
+            "build-commands": [
+                "sh make.sh --cflags=-DFLATPAK",
+                "make install PREFIX=/app"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/Alcaro/Flips.git"
+                }
```
Comment (nedrichards) [inline com.github.Alcaro.Flips.json]: cool, so all we're waiting for here is a release/tag/blessed hash from upstream that includes the changes.

Comment (Kekun) [reply com.github.Alcaro.Flips.json]: Alcaro doesn't want to release this version before years and finds tagging a random commit a bit silly, he is fine with releasing it with us using the commit that is currently on master though, so I pinned the repo to that specific commit. Hoping it's good. :)

**Comment by LeandroStanger** [comment]
bot, build com.github.Alcaro.Flips

File: com.github.Alcaro.Flips.json (hunk 2 above)
Comment (nedrichards) [reply com.github.Alcaro.Flips.json]: if upstream considers that 'blessed' then sure.

**Pull request closed without merging**